src/html.rs

```diff
@@ -100,8 +107,8 @@ impl HtmlMsgParser {
         } else {
             parser.cid_to_data_recursive(context, &parsedmail).await?;
         }
-
-        Ok(parser)
+        parser.html += &mem::take(&mut parser.msg_html);
```

Comment:
This results in several html documents joined sequentially, Firefox displays it w/o any problems, but this should be checked in the Delta Chat apps as well

Reply:
Doesn't work in DC Desktop, only the first HTML is shown :( Will join them into one HTML document then

Reply:
Probably it's not feasible to join HTMLs into one anyway, one HTML can be sent by another MUA, another one generated locally f.e.